internal/server/query.go

```diff
@@ -23,4 +23,15 @@ const (
 		delete from worker_auth_ca_certificate
  		where state = @state;
 	`
+	workerAuthRewrapQuery = `
+		select distinct
+		auth.worker_key_identifier,
+		auth.controller_encryption_priv_key,
+		auth.key_id
+		from server_worker worker
+		inner join worker_auth_authorized auth
+		on auth.worker_id = worker.public_id
+		and auth.key_id = ?
+		where worker.scope_id = ?
```

Comment:
I don't know if this matters, but it's weird to me that the `key_id` comparison is part of the join and not the where clause. What do you think about
```suggestion
		on auth.worker_id = worker.public_id
		where worker.scope_id = ? and auth.key_id = ?
```

Also consider using indentation to make this a bit easier to read, something like
```go
workerAuthRewrapQuery = `
		select distinct
			auth.worker_key_identifier,
			auth.controller_encryption_priv_key,
			auth.key_id
		from
			server_worker 		worker
		inner join
			worker_auth_authorized 	auth
		on
			auth.worker_id = worker.public_id
		where
			worker.scope_id = ? and auth.key_id = ?
`
```

Reply:
so the way this was explained to me is that adding the the comparison on the join yields a better query plan, but I double checked that the query plans are the same, so I'll update the query and add the indents.